### PR TITLE
New README

### LINKED ISSUE #125: Tweak the README to clarify API features
* New README as an entry point that clarifies various features
* Move clicks explainer to another file
* Remove `Name probably needs bikeshedding` from the main [README](https://github.com/WICG/conversion-measurement-api/blob/main/README.md)
* Edit what's already at the beginning of the [README](https://github.com/WICG/conversion-measurement-api/blob/main/README.md), something along the lines of: `This README focuses on attribution reporting for click-through conversions, at an event's level. Extended functionality on top of this is being considered: [aggregated reports](link to AGGREGATE.md) and ...`.

LMK whether these changes sounds good; happy to open a PR.

### PR DESCRIPTION
**Pull request by maudnals**
Fixes #125

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,579 +1,64 @@
+# Attribution Reporting API
 
-Click Through Attribution Reporting API Explainer
-============
+_This document is an overview of the Attribution Reporting API proposal. The API supports clicks and views across event-level and aggregate reports. If you're looking specifically for the explainer for event-level click reports that used to live on this page, head over to [Attribution Reporting API for Click-Through](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md) instead._
```
Comment (johnivdel) [inline README.md:3]:
```suggestion
_This document is an overview of the Attribution Reporting API proposal. The API supports measurement of clicks and views with event-level and aggregate reports. If you're looking specifically for the explainer for event-level click reports that used to live on this page, head over to [Attribution Reporting API for Click-Through](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md) instead._
```

File: README.md [hunk 2]
```diff
@@ -1,579 +1,64 @@
...
-Motivation
-----------
+### Event-Level reports (clicks and views)
 
-Currently, the web ad industry measures conversions via identifiers they
-can associate across sites. These identifiers tie information about
-which ads were clicked to information about activity on the advertiser's
-site (the conversion). This allows advertisers to measure ROI, and for
-the entire ads ecosystem to understand how well ads perform.
+Attribute cross-site click-through or view-through conversions at an event's level. See details in the [Explainer: event-level reports for views](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_views.md]) and in the [Explainer: event-level reports for clicks](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md]).
```
Comment (johnivdel) [inline README.md:33]:
```suggestion
Attribute cross-site click-through or view-through conversions with reports at a per-event level. See details in the [Explainer: event-level reports for views](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_views.md]) and in the [Explainer: event-level reports for clicks](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md]).
```

File: README.md [hunk 3]
```diff
@@ -1,579 +1,64 @@
...
 
-`<a attributiondestination="[eTLD+1]" attributionsourceeventid="[64-bit unsigned integer]"
-attributionexpiry=[unsigned long long] attributionreportto="[origin]">`
+Attribute conversions occurring across distinct devices, i.e. as distinct web browser instances. See details in the [Explainer](https://github.com/WICG/conversion-measurement-api/blob/main/cross_device.md).
 
-Attribution source attributes:
+Implementation status: `🟤 not implemented yet (all browsers)`
 
--   `attributiondestination`: an origin whose eTLD+1 is where attribution will be triggered for this source. 
+## Documentation
```
Comment (johnivdel) [inline README.md:59]:
```suggestion
## External Documentation
```

File: README.md [hunk 4]
```diff
@@ -1,579 +1,64 @@
...
+  Both report types can be used simultaneously. They're complementary.
 
--   **Conversion**: The completion of a meaningful (advertiser specified) user action on the advertiser's web site by a user who has previously interacted with an ad from that advertiser.
+## API features (proposals)
 
--   **Event-level data**: Data that can be tied back to a specific low-level event; not aggregated
+🕙 Last updated: June 2021
 
--   **Click-through-conversion (CTC)**: A conversion attributed to an ad click
+This API is being incubated and developed in the open. All the features below are proposals under incubation. This list evolves over time.
```
Comment (johnivdel) [inline README.md:29]:
```suggestion
All the features below are proposals under incubation. This list evolves over time.
```

We call this out above already

**Review by johnivdel — APPROVED** [review]
Just a few minor comments

File: README.md (hunk 2 above)
Comment (johnivdel) [reply README.md:33]: Actually I am not so sure about my suggestion here.

File: event_attribution_reporting_views.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Event Attribution Reporting API Explainer
+# Attribution Reporting API for View-Through
```
Comment (maudnals) [inline event_attribution_reporting_views.md:1]: @johnivdel Is this OK?

File: event_attribution_reporting_clicks.md [hunk 6]
```diff
@@ -0,0 +1,580 @@
+
+# Attribution Reporting API for Click-Through
```
Comment (maudnals) [inline event_attribution_reporting_clicks.md:2]: @johnivdel Is this OK?

File: event_attribution_reporting_views.md (hunk 5 above)
Comment (johnivdel) [reply event_attribution_reporting_views.md:1]: We could consider doing "Attribution Reporting for View-Through Measurement" and "Attribution Reporting for Click-Through Measurement" which align slightly better with the other explainer names.

But these seem fine as well.

Comment (maudnals) [reply event_attribution_reporting_views.md]: Used your suggestion. Thank you!

File: event_attribution_reporting_clicks.md (hunk 6 above)
Comment (maudnals) [reply event_attribution_reporting_clicks.md]: Used @johnivdel's suggestion above

**Pull request merged**